Tools-Override/FrameworkTargeting.targets

```diff
@@ -4,10 +4,10 @@
   <ItemGroup>
     <TargetingPackDirs Include="$(RefPath)" />
     <AdditionalReferencePaths Include="@(TargetingPackDirs)" />
+    <Reference Condition="($(NuGetTargetMoniker.StartsWith('.NETStandard')) OR '$(IsReferenceAssembly)' != 'true') AND Exists('$(RefPath)netstandard.dll') AND '$(MSBuildProjectExtension)' != '.pkgproj'" Include="netstandard" />
```

Comment:
Could we please create a property that controls this? Something like Include/ExcludeDefaultReferences and set that to true/false based on these various conditions. I really find it difficult to parse this extremely long set of conditions.